combirepo/kickstart_parser.py

```diff
@@ -48,6 +48,24 @@ def get_repository_names(self):
                     names.extend(re.findall(r"--name=(\S+)", line))
             return names
 
+    def get_images_mount_points(self):
+        """
+        Gets the names of images and corresponding mount points
+        that are mentioned in the kickstart file.
+        """
+        with open(self.path, "r") as kickstart_file:
+            images_dict = {}
+            for line in kickstart_file:
+                if line.startswith("part /"):
+                    image = re.findall(r"--label=(\S+)", line)
```

Comment:
Is mountpoint part the only possible place of `--label` in .ks file?

Reply:
As I see in kickstart docs, it is. All mountpoint lines start with "part". There is a search among theese lines only.

Reply:
Okay, LGTM then